common/src/main/java/com/wynntils/features/user/ChatTabsFeature.java

```diff
@@ -133,6 +135,17 @@ public void onScreenRender(ScreenRenderEvent event) {
         chatScreen.commandSuggestions.render(event.getPoseStack(), event.getMouseX(), event.getMouseY());
     }
 
+    @SubscribeEvent
+    public void onChatScreenKeyTyped(ChatScreenKeyTypedEvent event) {
+        // We can't use keybinds here to not conflict with TAB key's other behaviours.
+        if (event.getKeyCode() != GLFW.GLFW_KEY_TAB) return;
```

Comment:
Will this not break autocompletion..?

Reply:
🤦 It will. What keybind should we use then?

Reply:
Can we use ctrl-tab? Or that will interfere with window-changing in Windows, probably... If that still is ctrl-tab, and has not changed to like windows-tab or anything.

PgUp/PgDn, perhaps? Or ctrl-pgup/pgdn, or ctrl- left/right arrow? I'm not sure if any of those are used in vanilla to edit the chat message line or scroll the chat history.

Reply:
Shift tab should work on windows.. I guess

Reply:
An alternative might be to allow switching of tabs using the tab key, as long as the message bar is empty. If it is, the tab completetion is useless anyway (just starts inserting names of players logged in on the server), and we can use it for tab switching. If you have started typing, we can let vanilla handle the tab character for autocompletion.

The drawback is of course that you can only change tab with the mouse once you've started typing.

Reply:
Set to shift-tab.

Reply:
Ooookay. I was just about to suggest that "whatever you chose, perhaps adding shift should rotate in the opposite direction".

What about alt-tab? Or wait, maybe that was the window changing thing on Windows?

Reply:
alt tab is a windows thing tok

Reply:
Sorry, I've been using a mac for so long I've forgotten. It is alt-tab that is used by Windows.

ctrl-tab is used by Firefox on mac, windows and linux to switch tab (and shift-ctrl-tab to switch backwards). It is also used in vscode. I think that's good enough of a "universal standard" for me -- i recommend we use it as well.

Reply:
Do not worry I also only use linux and max daily. But, I remember windows keybinds. For now.

Reply:
I realize I was a bit too quick with pushing this for you. I did not check this carefully enough. (Too tired, I should have gone to bed long time ago...)

I think we should change the "shift-tab" to "ctrl+tab", and also let "shift+ctrl+tab" circle backwards through the tabs. Can you fix it in a separate PR? (Sorry for messing up)

Reply:
I'll later